static/js/components/dashboard/CourseDescription_test.js

```diff
@@ -35,11 +37,12 @@ describe('CourseDescription', () => {
 
   it('shows the course title and a link to view the course on edX', () => {
     for (let status of ALL_COURSE_STATUSES) {
-      let course = findCourse(course => (
+      let course = findAndCloneCourse(course => (
         course.runs.length > 0 &&
         course.runs[0].status === status
       ));
       let firstRun = course.runs[0];
+      firstRun.enrollment_url = 'http://test.com';
```

Comment:
If you need an example URL for a test, can you use `http://example.com`? [That's explicitly what it's for.](http://example.com/)